### ISSUE TITLE
Test exclusion during certification no longer working (e.g. Cert ~testname)

### ISSUE THREAD

**Issue by Smitholi67** [cnti-testcatalog/testsuite#2048]
**Describe the bug**
Prior to release 1.2.0, it was possible to execute the cert command and excluding specific tests (e.g. Cert ~testname).  We have noticed that this no longer works. 

**To Reproduce**
Run either Cert or the new Cert Essential with a specific test that you would like to exclude (for example Cert ~oran_e2_connection). 

**Expected behavior**
The excluded test should not execute if working properly.

**Screenshots**

**Device (please complete the following information):**

- OS [e.g. Linux, iOS, Windows, Android]
- Distro [e.g. Ubuntu]
- Version [e.g. 18.04]
- Architecture [e.g. x86, arm]

**How will this be tested? aka Acceptance Criteria (optional)**

(optional: unnecessary for things like spelling errors and such)

Once this issue is address how will the fix be verified?

**Additional context**

---

NOTE: you can enable higher logging level output via the command line or env var. to help with debugging

```
# cmd line
./cnf-testsuite -l debug test

# make sure to use -- if running from source
crystal src/cnf-testsuite.cr -- -l debug test

# env var
LOGLEVEL=DEBUG ./cnf-testsuite test
```

Also setting the verbose option for many tasks will add extra output to help with debugging

```
crystal src/cnf-testsuite.cr test_name verbose
```

Check [usage documentation](https://github.com/cncf/cnf-testsuite/blob/main/USAGE.md) for more info about invoking commands and logging

**Comment by agentpoyo** [comment]
I confirmed this is broken. Perhaps the refactoring of the points.yml layout caused this to break. We'll need to investigate.

**Comment by martin-mat** [comment]
it is broken by this PR
https://github.com/cnti-testcatalog/testsuite/pull/2012

in order to make it working again, an alternative approach needs to be implemented (exclusion list as a parameter?).
Or change current implementation of cert task.

**Comment by martin-mat** [comment]
Functionality now available as implemented in #2051 . Note that the usage is different now.
